PhysicsTools/PatAlgos/python/slimming/puppiForMET_cff.py

```diff
@@ -15,60 +15,29 @@ def makePuppies( process ):
                                            src = cms.InputTag("particleFlow"),
                                            pdgId = cms.vint32(-11,11,-13,13),
                                            )
-
-
     process.puppiNoLep = process.puppi.clone()
     process.puppiNoLep.candName = cms.InputTag('pfNoLepPUPPI') 
 
     process.puppiMerged = cms.EDProducer("CandViewMerger",src = cms.VInputTag( 'puppiNoLep','pfLeptonsPUPPET'))
-
-## puppi met
-    process.puppiForMET = cms.EDProducer("PuppiPhoton",
-                                         candName       = cms.InputTag('packedPFCandidates'),
-                                         puppiCandName  = cms.InputTag('puppi'),
-                                         photonName     = cms.InputTag('slimmedPhotons'),
-                                         photonId       = cms.InputTag("egmPhotonIDs:cutBasedPhotonID_PHYS14_PU20bx25_V2p1-standalone-loose"),
-                                         pt             = cms.double(10),
-                                         useRefs        = cms.bool(True),
-                                         dRMatch        = cms.vdouble(10,10,10,10),
-                                         pdgids         = cms.vint32 (22,11,211,130),
-                                         weight         = cms.double(1.),
-                                         useValueMap    = cms.bool(False),
-                                         weightsName    = cms.InputTag('puppi'),
-                                         )
-    process.puppiForMET.puppiCandName    = 'puppiMerged'
+    process.load('CommonTools.PileupAlgos.PhotonPuppi_cff')
+    process.puppiForMET = process.puppiPhoton.clone()
+    #process.puppiForMET.puppiCandName    = 'puppiMerged'
```

Comment:
@slava77 @violatingcp 
From Phil I understand that the puppi sequence will be complete when we are able to uncomment that line. So is just a memo for us.
As Phil wrote we have problem in mapping reference: we need reference for photons since they get weight zero and leptons since they eat pileup.